samples/go/nomsfs/README.md

```diff
@@ -1,29 +1,39 @@
+# nomsfs
 
-# Getting Started on Mac OS X
+Nomsfs is a [FUSE](https://en.wikipedia.org/wiki/Filesystem_in_Userspace) filesystem built on Noms. To use it you'll need FUSE:
+
+* *Linux* -- built-in; you should be good to go
+* *Mac OS X* -- Install [FUSE for OS X](https://osxfuse.github.io/)
+
+Development and testing have been done exclusively on MacOS X using FUSE OS X.
+Nomsfs builds on the [Go FUSE imlementation](https://github.com/hanwen/go-fuse) from Han-Wen Nienhuys.
+
+## Usage
+
+Make sure FUSE is installed. On macOS remember to run `/Library/Filesystems/osxfusefs.fs/Support/load_osxfusefs`.
+
+
+Build with `go build`; test with `go test`.
+
+Mount an existing or new dataset by executing `nomsfs`:
 
-1. Install [FUSE for OS X](https://osxfuse.github.io/)
-2. Load FUSE for OS X:
-```
-> sudo /Library/Filesystems/osxfusefs.fs/Support/load_osxfusefs
-```
-3. Create a local database:
-```
-> noms serve ldb:/tmp/nomsfs &
-```
-4. Build and run `nomsfs`:
 ```
-> go run nomsfs.go http://localhost:8000::nomsfs directoryForMount
+$ mkdir /var/tmp/mnt
+$ go run nomsfs.go /var/tmp/nomsfs::fs /var/tmp/mnt
```

Comment:
You said `go build` and `go test` above, if already built, then just `$ ./nomsfs` rather than `go run nomsfs.go`